app/src/app/rc.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import {app} from 'electron'
+import {existsSync} from 'fs'
+import loadJson from 'load-json-file'
+import {isWindows, isOSX} from './utils'
+
+// Directory where of .app or .exe file lives.
+let dir
+
+if (isOSX()) dir = path.normalize(app.getAppPath() + '/../../../..')
```

Comment:
Sorry, didn't mention there, for consistency we decided to put this file on all systems near the .exe or .app file.